src/components/landing/Heading.svelte

```diff
@@ -79,10 +79,10 @@
   <div class="text-container">
     <DottedAccent top="25vh" left="60%" width="l" height="m" />
     <img src={content.logo.src} alt={content.logo.alt} />
-    <h2>
+    <h1>
```

Comment:
Semantically, should this be an `<h1>`? Perhaps the `<img>` should be wrapped in an `<h1>` and this should remain an `<h2>`. Thoughts?

Reply:
Very arguable which to use. You want a readable `h1` on the homepage for SEO, and an image isn't quite the same. So, decided to make this guy the `h1`

Reply:
Linked some of these to you over other channels, but dropping them here for reference:  
https://moz.com/community/q/is-alt-text-inside-an-img-tag-inside-an-h1-the-same-weight-as-text-directly-inside-the-h1  
https://stackoverflow.com/a/669847/5233414  
https://www.youtube.com/watch?v=fBLvn_WkDJ4

So, I would still go for the `<h1>` around the image.